src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -89,7 +89,9 @@ public void storeMessage(Message message) {
         long timestamp = (long) entity.getProperty("timestamp");
         String text = (String) entity.getProperty("text");
         String imageUrl = (String) entity.getProperty("imageUrl");
-        String imageLabels = (String) entity.getProperty("imageLabels");
+        
+        @SuppressWarnings("unchecked")
```

Comment:
Is there a better way to cast `List<Object>` as `List<String>`?

Check out:
1. https://stackoverflow.com/questions/1917844/how-to-cast-listobject-to-listmyclass/1920865
2. https://stackoverflow.com/questions/933447/how-do-you-cast-a-list-of-supertypes-to-a-list-of-subtypes